docs/apache-airflow/concepts/dags.rst

```diff
@@ -177,6 +179,20 @@ In much the same way a DAG instantiates into a DAG Run every time it's run,
 Tasks specified inside a DAG are also instantiated into
 :ref:`Task Instances <concepts:task-instances>` along with it.
 
+When the DAG run starts, it will have the start date and end date. This period
+describes the time when the DAG run actually 'ran.' Aside from the DAG
+run's start and end date, there is another date called ``logical date``
+(formally known as execution date), which describes the intended time for the
+DAG run to be scheduled or triggered to run. The reason why this is called
+``logical`` is because of the abstract nature of it having multiple meanings,
+depending on the context of the DAG run itself.
```

Comment:
```suggestion
run's start and end date, there is another date called *logical date*
(formally known as execution date), which describes the intended time a
DAG run is scheduled or triggered. The reason why this is called
*logical* is because of the abstract nature of it having multiple meanings,
depending on the context of the DAG run itself.
```

Double backticks are for code, not emphasis. (same applies for other parts below that use backticks)

Also, perhaps it’s better to use present tense here? I feel future tense hints that this time is toward the _end_ of the data interval, which is not always the case. (Or maybe let’s remove the “which describes” part altogether? The most important takeaway here is that “logical date” is abstract and have multiple meanings, and everything else can be potentially misleading and dilute the thing we want to get across.)